public-api/src/main/java/org/pmiops/workbench/google/GoogleAnalyticsService.java

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.google;
+
+import java.io.IOException;
+
+public interface GoogleAnalyticsService {
+
+    int trackEventToGoogleAnalytics(
+            String cid, String category, String action, String label, String value)
```

Comment:
opt: may want `@Nullable`  on cid to make this clear

Reply:
Done